Documentation/config/windows.txt

```diff
@@ -0,0 +1,4 @@
+windows.appendatomically::
```

Comment:
Please use `appendAtomically` in the documentation, it is easier to read.

Reply:
sorry, I missed it, let me try.